HW Assignment #1/About_Me.html

```diff
@@ -0,0 +1,26 @@
+
+
+
+<link rel="stylesheet" type="text/css" href="style.css">
+
+<center>
+	<h1> TASNEEM KARIM</h1>
+
+
+<center>
+	<img src= "Grand Canyon.jpg" width="95" height="95"/ alt="Grand Canyon pic"> 
```

Comment:
Spacing won't work with files. To replicate a space use the underscore ( _ ) character. Nice work using the alt attribute.